src/Base/Unit.cpp

```diff
@@ -656,6 +659,7 @@ const Unit Unit::ElectricalInductance       (2, 1, -2, -2);
 const Unit Unit::ElectricalResistance       (2, 1, -3, -2);
 const Unit Unit::ElectricCharge             (0, 0, 1, 1);
 const Unit Unit::ElectricPotential          (2, 1, -3, -1);
+const Unit Unit::ElectricFlux               (-2, 0, 1, 1);
```

Comment:
`m^-2 * 1 * s * A`, looks a lot like Surface Charge Density, isn't ElectricFlux: `Vm` = `m^3 * kg * s^-3 * A^-1` which should be `(3, 1, -3, -1)`?

Reply:
I agree that the naming of the unit is a bit unconventional. Because this module is an elmer wrapper, I decided to follow the nomenclature of Elmer. If you check out the explanation in the [Elmer Models Manual](https://www.nic.funet.fi/index/elmer/doc/ElmerModelsManual.pdf) (page 111 for the boundary conditions and page 113 for the sif-file keyword listing), you'll see that the _"Electric Flux"_ is specified in terms of the (free) surface charge density.

Reply:
This stuff is a bit over my head but to me, it sounds like a special condition must be met for `electric flux` to be equal to `surface charge density`, so in their case it is true, but it might not be equal in other cases (which would explain the difference in base units).

Even if I'm wrong, wouldn't it be better to keep a conventional name in FreeCAD's core, and only make the translation to electric flux (if needed) in the elmer integration?

Reply:
@marioalexis84, I'll change the unit to _surface charge density_; should the name be `Unit::ElectricSurfaceChargeDensity` or `Unit::SurfaceChargeDensity`?
This is the _"free"_ surface charge density related to the electric displacement field **D** (ref. e.g. Introduction to Electrodynamics -- D.J. Griffiths - Prentice Hall). Should that be reflected in the naming, or is just plain "SurfaceChargeDensity" sufficient?

Reply:
`SurfaceChargeDensity` is sufficient. It doesn't matter if it comes from free charges or from polarization charges, both are charge densities.